backend_new/src/services/application-csv-export.service.ts

```diff
@@ -158,27 +157,10 @@ export class ApplicationCsvExporterService {
                   return preference;
                 }
 
-                if (parsePrograms) {
-                  // curr should equal the program id we're pulling from
-                  if (!programs) {
-                    programs = JSON.parse(app.programs as string);
-                  }
-                  parsePrograms = false;
-                  const program = programs.find(
-                    (preference) => preference.multiselectQuestionId === curr,
-                  );
-                  multiselectQuestionValue = true;
-                  return program;
-                }
```

Comment:
I believe all the programs we have are added manually, we don't actually loop through programs in this way to add them to the CSV the same way we do with preferences since they can be changed per listing.